app/src/main/res/xml/preferences.xml

```diff
@@ -36,6 +36,14 @@
             android:summary="%s"
             android:title="@string/pref_status_text_size" />
 
+        <ListPreference
```

Comment:
I went for a ListPreference instead of a SwitchPreference because it makes it clearer that there are two exclusive options imho.